cypress/integration/userCanSearchDrinks.feature.js

```diff
@@ -0,0 +1,38 @@
+describe("User can search", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("http://localhost:3001");
+  });
+  it("search field exists", () => {
+    cy.get("#name-search.prompt").should("exist");
+    cy.get("#search").should("exist");
+    cy.get("#search").contains("Search");
+  });
```

Comment:
I don't see this file as being really relevant or at least to be included in this file. Remove or extract to a separate test, please

Reply:
We have removed that part now.